shared/secrets_masker/tests/secrets_masker/test_secrets_masker.py

```diff
@@ -779,6 +820,10 @@ def test_mixed_structured_unstructured_data(self):
 class TestSecretsMaskerMerge:
     """Test the merge functionality for restoring original values from redacted data."""
 
+    def setup_method(self):
+        self.masker = SecretsMasker()
+        configure_secrets_masker_for_test(self.masker)
```

Comment:
`merge` doesnt really need separate instance of masker per test as it doesn't change state of it